src/components/screens/chat/chatFormBlocks/ChatFormInputs.js

```diff
@@ -131,35 +134,61 @@ export default function ChatFormInputs({
 
   // vv  Attachments pick  vv //
   const pickUserFiles = () => filePicker.current.click();
-  const handlerChange = (event) => {
+  const handlerChange = async (event) => {
     if (!event.target.files.length) {
       return;
     }
+    setIsLoadingFile(true);
 
     const selectedFiles = [];
-    for (const file of event.target.files) {
-      if (file.name.length > 255) {
-        showCustomAlert(
-          "The file name should not exceed 255 characters.",
-          "warning"
-        );
-        return;
-      }
-      if (!file.type.startsWith("image/")) {
-        showCustomAlert("Please select an image file.", "warning");
-        return;
-      }
+    try {
+      for (const file of event.target.files) {
```

Comment:
use forEach array's method instead
const files = event.target.files;
files.forEach(file)

Reply:
But files are Object, so forEach is not the best way to read them, I suggest you use `for`

https://stackoverflow.com/questions/25052758/for-in-vs-object-keys-foreach-without-inherited-properties